src/components/pool_flux_chart.vue

```diff
@@ -105,13 +139,13 @@ export default {
           this.drawChart(this.volume, 1)
 
           // set up radio button interaction
-          this.chartContainer.selectAll("input").on("click", this.changeXScale.bind(this));
+          this.captionContainer.selectAll("input").on("click", this.changeXScale.bind(this));
           
         },
         adaptScales(data, xMin) {
           Object.keys(this.scales).forEach(function (scaleType) {
               this.scales[scaleType]
-                  .domain([xMin, this.d3.max(data, d => d.value_km_3)])
+                  .domain([xMin, this.d3.max(data, d => d.range_high)])
```

Comment:
The axis labels for linear are overlapping for me
![image](https://user-images.githubusercontent.com/17803537/190801358-348bdbe6-28a0-41ef-a1ef-31bb6311b0d4.png)

Reply:
We should use labels for either millions or billions, e.g. 100 M or 0.1 B

Reply:
Yeah - I should tweak those, too.